doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=[Mesh] end=This assembles the 7 pins into an assembly with a duct include-end=False
+
+These pin types are used to build the assembly with duct using `HexIDPatternedMeshGenerator`. Each pin type represents fuel pins for each hexagonal ring. `HexIDPatternedMeshGenerator` is meant to assign some extra element integer ID, here `pin_id`, for differentiating pins. It should be noted that `background_block_id` needs to be different from `background_block_ids` for a pin in `PolygonConcentricCircleMeshGenerator` in order to properly generate `pin_id`, which is why `bid_F_lead + 1` is used instead of `bid_F_lead` in `background_block_id`. Note that duct and assembly lead gap regions are built using `duct_block_ids`, `duct_sizes` and `duct_intervals`.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=This assembles the 7 pins into an assembly with a duct end=[] include-end=True
+
+Before extruding the 2D assembly geometry built above, a name of the duct and lead interface is explicitly defined.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=interface_define end=[] include-end=True
+
+The 2D assembly is now extruded to the z-direction and new block IDs are assigned.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=This extrudes the 7-pin assembly axially and renames block IDs appropriately end=[] include-end=True
+
+Sidesets are renamed with common sensical names.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=This renames sidesets to common sensical names end=[] include-end=True
+
+Material IDs are assigned to block IDs.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=[assign] end=[] include-end=True
+
+A coarse mesh is generated for the CMFD acceleration. For better alignment of peripheral pin meshes near duct, only the peripheral pins have two radial meshes divided by the cladding outer radius while inner pins have one radial mesh. Azimuthal divisions are the same as the fine mesh. The duct mesh is also the same as the fine mesh, otherwise the CMFD solve didn't converge. The axial mesh is also the same as the fine mesh because the use of a coarser axial mesh increased the number of transport sweeps significantly. 
```

Comment:
Coarser duct mesh can cause CMFD divergence? You mean CMFD solver diverges but not Richardsion iteration, right?

Reply:
CMFD solve diverged, not Richardson iteration.